dask/distributed/client.py

```diff
@@ -113,5 +113,17 @@ def get_collection(self, name):
 
         return payload2['type'](*payload2['args'])
 
-    def close(self):
+    def close_workers(self):
+        header = {'function': 'close-workers'}
+        self.send_to_scheduler(header, {})
+
+    def close_scheduler(self):
+        header = {'function': 'close'}
+        self.send_to_scheduler(header, {})
+
+    def close(self, close_workers=False, close_scheduler=False):
+        if close_workers:
+            self.close_workers()
+        if close_scheduler:
+            self.close_scheduler()
```

Comment:
Suggest keeping the close and close_scheduler functions completely separate (no kwargs).  Suggest removing the close_workers function and instead have the scheduler shut down workers by default when it closes.